src/server/status_server/mod.rs

```diff
@@ -673,6 +680,9 @@ where
                             (Method::GET, "/debug/pprof/profile") => {
                                 Self::dump_cpu_prof_to_resp(req).await
                             }
+                            // (Method::GET, "/debug/pprof/heap") => {
+                            //     Self::dump_heap_prof_to_resp(req).await
+                            // }
```

Comment:
remove?

Reply:
We are considering to factor it and maybe add it back, so keeping this comment would be better.